app/models/manageiq/providers/terraform_enterprise/automation_manager/orchestration_stack/status.rb

```diff
@@ -0,0 +1,52 @@
+class ManageIQ::Providers::TerraformEnterprise::AutomationManager::OrchestrationStack::Status < ::OrchestrationStack::Status
+  def initialize(status)
+    self.status = status
+    self.reason = self.class.run_state_to_description(status)
+  end
+
+  def succeeded?
+    status.downcase == "planned_and_finished"
+  end
+
+  def failed?
+    %w[policy_soft_failed errored].include?(status.downcase)
+  end
+
+  def canceled?
+    %w[discarded canceled force_canceled].include?(status.downcase)
+  end
+
+  # Lookup table of run state descriptions
+  # https://developer.hashicorp.com/terraform/cloud-docs/api-docs/run#run-states
+  def self.run_state_to_description(status)
+    case status
+    when "pending"              then N_("Pending")
```

Comment:
Since you used `N_` here, is something else in a different layer going to do the actual translation?

Reply:
Is there downside to using `n_()` if this is backend-only?  I am not sure how the UI displays this but didn't want to miss out on translating these strings

Reply:
`n_` is not right because that's for pluralization. There's no downside to using `N_` here other than the obvious that it gets put in the translation catalog and needs translating.